src/Sylius/Component/Metadata/spec/Model/Twitter/PlayerCardSpec.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Component\Metadata\Model\Twitter;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Metadata\Model\MetadataInterface;
+use Sylius\Component\Metadata\Model\Twitter\PlayerCard;
+
+/**
+ * @mixin \Sylius\Component\Metadata\Model\Twitter\PlayerCard
+ *
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class PlayerCardSpec extends ObjectBehavior
+{
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Component\Metadata\Model\Twitter\PlayerCard');
+    }
+
+    function it_implements_Summary_Card_interface()
+    {
+        $this->shouldImplement('Sylius\Component\Metadata\Model\Twitter\PlayerCardInterface');
+    }
+
+    function it_is_serializable()
+    {
+        $that = new PlayerCard();
+        $that->setTitle('Lorem ipsum');
+
+        $that = unserialize(serialize($that));
+
+        \PHPUnit_Framework_Assert::assertEquals('Lorem ipsum', $that->getTitle());
+    }
+
+    function it_is_mergable_with_same_class_object()
+    {
+        $metadata = new PlayerCard();
```

Comment:
That's why I don't like magic... It's still better than no test, so let's leave it as it is now.